src/qaqc/download_and_flag_fxns/add_seasonal_flag.R

```diff
@@ -23,7 +23,12 @@ add_seasonal_flag <- function(df) {
     # flag obs whose slope is greater than the 99th percentile range
     add_flag((slope_ahead >= t_slope_behind_99 | slope_behind >= t_slope_behind_99), "slope violation") %>%
     # ... flag obs that are outside of the rolling mean times 3 sd's of the 1-99 percentile seasonal values
-    add_flag((mean <= rollavg - (3 * t_sd_0199) | mean >= rollavg + (3 * t_sd_0199)), "outside sd range")
+    add_flag((mean <= rollavg - (3 * t_sd_0199) | mean >= rollavg + (3 * t_sd_0199)), "outside sd range") %>%
+    # ... flag obs that are outside of the rolling mean times 3 sd's of the 1-99 percentile seasonal values
+    add_flag((mean <= rollavg - (3 * t_sd_0199) | mean >= rollavg + (3 * t_sd_0199)), "outside sd range") #%>%
```

Comment:
It looks like this flag is being added twice.

Reply:
good catch!